### DISCUSSION TITLE
How to get package once requested thorugh PR

### DISCUSSION THREAD

**Discussion by joerowelll** [RoboStack/ros-noetic#377] [Q&A] [answered]
I have requested octomap-ros thorugh [this PR](https://github.com/RoboStack/ros-noetic/pull/376#event-9768620140), and it has been merged into master. I have the following channels in my mamba env:
- conda-forge/linux-64                                        
- conda-forge/noarch                               
- robostack-staging/linux-64                            
- robostack-staging/noarch               
However, when I now try and install ros-noetic-octomap-ros, I still get the error "  - nothing provides requested ros-noetic-octomap-ros".
What steps am I missing?

**Comment by joerowelll** [comment]
As far as I can see, it has been added to all architectures but linux-64 https://robostack.github.io/noetic.html , although the PR that merged added octomap-ros to linux-64.

**Comment by Tobias-Fischer** [comment, marked as answer]
It should work now :)

**Reply by joerowelll** [reply]
Thank you
